### ISSUE TITLE
[Plus 1.1.3] Issue related to #1935 Double Click behaviour

### ISSUE THREAD

**Issue by RandomGOTI** [sandboxie-plus/Sandboxie#1951]
### What happened?

Double Clicking the Path cell triggers 2 actions for me  , will open explorer but also the sandbox options

Note:Holding down control key and clicking on the path cell will not open explorer but sandbox options as previously suggested
Looking at the code , wouldn't adding a `return;` like this fix the issue of having 2 actions?

```
if ((QGuiApplication::queryKeyboardModifiers() & Qt::ControlModifier) == 0) {

		if (index.column() == CSbieModel::ePath)
			OnSandBoxAction(m_pMenuExplore);
			return;
	}

```

### Download link

about:blank

### To Reproduce

1 any non empty box
2 double click on its path cell
3 both explorer and its sandbox options window show

### Expected behavior

Double Clicking the Path cell triggers only 1 action (opening explorer) if control is hold down it will open the sandbox options instead

### What is your Windows edition and version?

Windows 7 Ultimate SP1 x64

### In which Windows account you have this problem?

Not relevant to my request.

### Please mention any installed security software

None/WD disabled

### What version of Sandboxie are you running?

Plus 1.1.2 x64

### In which sandbox type you have this problem?

Not relevant to my request.

### Where is the program located?

Not relevant to my request.

### Can you reproduce this problem on an empty sandbox?

Not relevant to my request.

**Comment by RandomGOTI** [comment]
Plus 1.1.3 x64 Still opens Sandbox Options aswell when just double clicking the path

**Comment by DavidXanatos** [comment]
indeed the fix did  not make it to the git branch in time, will be in included in 1.2.0
